src/App.tsx

```diff
@@ -15,71 +15,78 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [selectedGood, setSelectedGood] = useState<string | null>('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const resetState = () => {
+    setSelectedGood(null);
+  };
```

Comment:
Set empty string here if there are no selected elements, empty will return false if convert to boolean and there will be always string type value in your state